pkg/netflow/goflowlib/metric.go

```diff
@@ -66,7 +66,7 @@ var metricNameMapping = map[string]mappedMetric{
 		},
 	},
 	"flow_process_nf_count": {
-		name:           "processor.flows",
+		name:           "processor.processed",
```

Comment:
Note: Renamed `datadog.netflow.processor.flows` to `datadog.netflow.processor.processed` since we are not measuring the number of flows but UDP packets being processed